ext/json/json.c

```diff
@@ -95,6 +96,9 @@ static PHP_MINIT_FUNCTION(json)
 	INIT_CLASS_ENTRY(ce, "JsonSerializable", json_serializable_interface);
 	php_json_serializable_ce = zend_register_internal_interface(&ce);
 
+	INIT_CLASS_ENTRY(ce, "JsonException", NULL);
```

Comment:
@hikari-no-yume I don't see any value in renaming that. The RFC was just for the future scope so it certainly doesn't apply for `JsonSerializable`. It's already documented and people are used to that name so I wouldn't change it. I think that such change needs a separate RFC.